tests/test_0001_html.py

```diff
@@ -336,3 +337,46 @@ def test_html_strings():
     node = HTML('<button disabled="true">Click me</button>')[0]
 
     assert node.disabled
+
+    # data binding nodes ######################################################
+    node = HTML('<input value="abc"/>')[0]
+
+    assert type(node) is TextInput
+    assert node.value == 'abc'
+
+    node = HTML('<input type="text" value="xyz" disabled/>')[0]
+
+    assert type(node) is TextInput
+    assert node.value == 'xyz'
+    assert node.disabled == True  # NOQA: E712
+
+    node = HTML('<input type="number" value="123"/>')[0]
+
+    assert type(node) is Node
+
+    node = HTML('<input type="checkbox"/>')[0]
+
+    assert type(node) is CheckBox
+    assert node.value == False  # NOQA: E712
+
+    node = HTML('<input type="checkbox" checked/>')[0]
+
+    assert type(node) is CheckBox
+    assert node.value == ''
+    # assert node.value == True  # TODO: fix in the next PR
```

Comment:
Is this broken? This should be an boolean.

Reply:
Yes it is broken. I will create PR with fix after this one.

Reply:
#84